account_multicurrency_revaluation/model/account.py

```diff
@@ -80,14 +77,13 @@ def compute_revaluations(
         # compute for each account the balance/debit/credit from the move lines
         ctx_query = context.copy()
         ctx_query['periods'] = period_ids
```

Comment:
Using `with_context()` you can specify a key which will be added to self.env.context

    self.with_context(periods=period_ids)...